<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTcpServer</class>
 <widget class="QDialog" name="DialogTcpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>服务器参数</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP地址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serverIPList"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverPort">
        <property name="inputMask">
         <string>DDDDD; </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnListen">
        <property name="text">
         <string>监听</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>客户端状态</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IP地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="clientIP">
        <property name="inputMask">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>端口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="clientPort">
        <property name="inputMask">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>serverIPList</zorder>
  <zorder>label_2</zorder>
  <zorder>serverPort</zorder>
  <zorder>btnListen</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <tabstops>
  <tabstop>serverIPList</tabstop>
  <tabstop>serverPort</tabstop>
  <tabstop>btnListen</tabstop>
  <tabstop>clientIP</tabstop>
  <tabstop>clientPort</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
